module5/labs/Week1/RefactoringPatterns.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Refactoring Patterns Lab
+---
+
+## Disordered Orders
```

Comment:
I'm very excited for this lab!! It's some pretty meaty new code for students to dive into, and I'm sure it took a bit to create even with the help of ChatGPT. I especially like how you added comments 👏